cmd/aigw/run_test.go

```diff
@@ -59,15 +63,16 @@ func TestRun(t *testing.T) {
 	}()
 	defer func() { cancel(); <-done }()
 
+	client := openai.NewClient(option.WithBaseURL("http://localhost:1975/v1/"))
+	chatReq := openai.ChatCompletionNewParams{
+		Messages: []openai.ChatCompletionMessageParamUnion{
+			openai.UserMessage("Say this is a test"),
+		},
+		Model: ollamaModel,
+	}
 	t.Run("chat completion", func(t *testing.T) {
```

Comment:
@mathetake for me, not sure about CI, but I routinely cannot get this to pass on main or this branch as the config always ends up as a 503 error. In docker it is fine 🤷